doc/classes/SpatialMaterial.xml

```diff
@@ -24,6 +25,7 @@
 			<argument index="0" name="flag" type="int" enum="SpatialMaterial.Flags">
 			</argument>
 			<description>
+				Returns [code]true[/code], if the specified [enum Flags] is enabled.
```

Comment:
Works for `[Feature]`, but less for the plural `[Flags]`. Should probably be "if the specified flag is enabled. See [enum Flags] for options."